### ISSUE TITLE
Packages for units & numbers

### ISSUE THREAD

**Issue by gonuke** [CNERG/templates#7]
Some of these already may be in templates, but it might be useful to seek out packages that help us normalize the way we write units and scientific numbers.

**Comment by kkiesling** [comment]
yes- the package `siunitx` is included in the preamble for pr #6 specifically for this purpose (example numbers and units also included in the example text of that pr).

**Comment by gonuke** [comment]
I wonder if could have a file that folks can include in any document that might not rely on these templates that incorporates a list of recommended packages....
